src/commands/info/help.ts

```diff
@@ -47,23 +54,29 @@ export default class extends Command {
 				.setTitle(command.name)
 				.setDescription(command.description ? command.description : '')
 				.addFields(fields)
-				.setThumbnail(msg.client.user.avatarURL())
+				.setThumbnail(interaction.client.user.avatarURL())
 				.setTimestamp(Date.now())
 				.setColor('RANDOM');
 
-			return msg.channel.send({ embeds: [embed] });
+			return interaction.reply({ embeds: [embed] });
 		} else {
-			let helpStr = `You can do \`${PREFIX}help <command>\` to get more information about any command, or you can visit our website here:\n<${website}>\n`;
+			// if no command given
+			let helpStr = `You can do \`/help <command>\` to get more information about any command, or you can visit our website here:\n<${website}>\n`;
 			const categories: Array<string> = [];
 			commands.forEach(command => {
 				if (!categories.includes(command.category)) categories.push(command.category);
 			});
 
+			const member = interaction.member as GuildMember;
+			const staff = interaction.guild.roles.cache.find(r => r.name === 'Staff');
 			categories.forEach(cat => {
-				const useableCmds = commands.filter(command =>
+				let useableCmds = commands.filter(command =>
 					command.category === cat
-					&& !(command.permissions && !command.permissions(msg))
 					&& command.enabled !== false);
+				// check if user isn't staff
+				if (!member.roles.cache.has(staff.id)) {
```

Comment:
Remember there are admin specific commands too, make sure to have a conditional for this too